tsconfig.base.json

```diff
@@ -2,6 +2,8 @@
 	"compilerOptions": {
 		"composite": true,
 		"declaration": true,
+		"declarationMap": true,
+		"sourceMap": true,
```

Comment:
The former fixes jump-to-definition and the latter lets the VS Code debugger function. I figure these are fine to keep? (maybe send in as a separate PR as well so they can get into `main`?)